src/snowflake/snowpark/modin/pandas/resample.py

```diff
@@ -417,9 +417,25 @@ def prod(
         # TODO: SNOW-1063368: Modin upgrade - modin.pandas.resample.Resample
         self._method_not_implemented("prod")
 
-    def size(self):  # pragma: no cover
+    def size(self):
         # TODO: SNOW-1063368: Modin upgrade - modin.pandas.resample.Resample
-        self._method_not_implemented("size")
+        from .series import Series
+
+        is_series = not self._dataframe._is_dataframe
```

Comment:
Mostly copied from Modin, but Modin has defined methods for each resample aggregation (i.e. `resample_size`, `resample_min`). Will refactor the frontend in another PR.